components/order-list.jsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import Image from 'next/image';
+import { BoxRegularIcon, WeightIcon } from '@/icons';
+import { formatDate } from '@/lib/utils';
+import { ORDER_STATUS_TYPE } from '@/lib/constants';
+import { useRouter } from 'next/router';
+import Checkbox from './checkbox';
+import OrderStatusBadge from './order-status-badge';
+
+const OrderItem = ({ order }) => {
+  const statusType = ORDER_STATUS_TYPE[order.status];
+  const { push } = useRouter();
+  const handleNavigation = () => {
+    push(`/orders/${order.id}`);
+  };
+  return (
+    <tr
+      onClick={handleNavigation}
+      className="text-gray-800 border-t border-gray-200 cursor-pointer text-start hover:bg-gray rounded-xl"
+    >
+      <td className="p-2 min-w-max">
+        <Checkbox label="Select item" />
+      </td>
+      <td className="p-2">
+        <div className="min-w-max"> {order.id}</div>
+      </td>
+      <td className="p-2 min-w-max">
+        <ItemShortDetails itemDetails={order.item} label="Item short details" />
+      </td>
+      <td className="p-2">
+        <div className="min-w-max">{order.item.type}</div>
+      </td>
+      <td className="p-2">
+        <div className="min-w-max"> {order.total_quantity}</div>
+      </td>
+      <td className="p-2">
+        <div className="min-w-max"> {`${order.total_weight} (${order.unit})`} </div>
+      </td>
+      <td className="p-2">
+        <OrderStatusBadge label={statusType.badgeTitle} variant={statusType.color} />
+      </td>
+      <td className="p-2">
+        <div className="min-w-max"> {formatDate(order.expected_delivery_date, false)}</div>
+      </td>
+    </tr>
+  );
+};
+
+const ItemShortDetails = ({ itemDetails }) => {
```

Comment:
We can reuse this in the order details page. So creating a separate component file should be better.